### PR TITLE
Issue template cleanup

### PR DESCRIPTION
**Pull request by david-waltermire-nist**
Updated issue templates using GitHub forms.
Also added contributing and code of conduct documentation.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-feedback.yaml [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+      placeholder: |
+        Provide your feedback here.
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Where, exactly?
+      description: Provide some context about what this feedback relates to.
+      placeholder: |
+        - For the registry, please indicate the tab and cell, or other clear identifier
```
Comment (Rene2mt) [inline .github/ISSUE_TEMPLATE/1-feedback.yaml:53]: We should have users provide the version number that is impacted by the issue or defect they are reporting.

File: .github/ISSUE_TEMPLATE/3-action-item.yaml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      - fix - something needs to be different
+      - improvement - something could be better
+      - research - something needs to be investigated
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: This relates to ...
+      description: Select all things this feedback relates to.
+      options:
```
Comment (Rene2mt) [inline .github/ISSUE_TEMPLATE/3-action-item.yaml:26]: Similar to previous comment, consider adding options for the baselines and "other".

File: .github/ISSUE_TEMPLATE/2-bug-report.yaml [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    attributes:
+      label: How do we replicate this issue?
+      description: What are the steps to reproduce this behavior (use screenshots if applicable)?
+      placeholder: |
+        1. Do this...
+        2. Then this...
+        3. See error...
+    validations:
+      required: true
+  - type: textarea
```
Comment (Rene2mt) [inline .github/ISSUE_TEMPLATE/2-bug-report.yaml:59]: Should we add an entry where users can provide details on where they identified the issue?  Where could be version content or tools, URL link to source code in repo, etc.  Here's what we have in the feedback form:

```
  - type: textarea
    attributes:
      label: Where, exactly?
      description: Provide some context about what this feedback relates to.
      placeholder: |
        - For the registry, please indicate the tab and cell, or other clear identifier
```

File: .github/ISSUE_TEMPLATE/1-feedback.yaml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+      - kudos - want to encourage something helpful
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: This relates to ...
+      description: Select all things this feedback relates to.
+      options:
+      - label: the **FedRAMP OSCAL Registry**
+      - label: the **Guide to OSCAL-based FedRAMP Content**
```
Comment (david-waltermire-nist) [inline .github/ISSUE_TEMPLATE/1-feedback.yaml:30]:
```suggestion
      - label: the **FedRAMP OSCAL baselines**
      - label: the **Guide to OSCAL-based FedRAMP Content**
```

File: .github/ISSUE_TEMPLATE/2-bug-report.yaml [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: checkboxes
+    attributes:
+      label: This relates to ...
+      description: Select all things this feedback relates to.
+      options:
+      - label: the **FedRAMP OSCAL Registry**
+      - label: the **Guide to OSCAL-based FedRAMP Content**
```
Comment (david-waltermire-nist) [inline .github/ISSUE_TEMPLATE/2-bug-report.yaml:23]:
```suggestion
      - label: the **FedRAMP OSCAL baselines**
      - label: the **Guide to OSCAL-based FedRAMP Content**
```

File: .github/ISSUE_TEMPLATE/1-feedback.yaml [hunk 6]
```diff
@@ -0,0 +1,63 @@
...
+      - concern - something needs to be different
+      - question - need to understand something
+      - kudos - want to encourage something helpful
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: This relates to ...
+      description: Select all things this feedback relates to.
+      options:
```
[the remark this answers is not in the document]
Comment (david-waltermire-nist) [reply .github/ISSUE_TEMPLATE/1-feedback.yaml:28]: added baselines below. I figured for "other" they could just not select a checkbox. Would it be better to have an explicit other?

File: .github/ISSUE_TEMPLATE/3-action-item.yaml [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+      - label: the **FedRAMP OSCAL Registry**
+      - label: the **Guide to OSCAL-based FedRAMP Content**
```
Comment (david-waltermire-nist) [inline .github/ISSUE_TEMPLATE/3-action-item.yaml:28]:
```suggestion
      - label: the **FedRAMP OSCAL baselines**
      - label: the **Guide to OSCAL-based FedRAMP Content**
```

File: .github/ISSUE_TEMPLATE/3-action-item.yaml (hunk 2 above)
Comment (david-waltermire-nist) [reply .github/ISSUE_TEMPLATE/3-action-item.yaml:26]: Fixed below.

**Pull request merged**